api/v1alpha1/registry_types.go

```diff
@@ -29,6 +31,9 @@ type RegistrySpec struct {
 	CABundle string `json:"caBundle,omitempty"`
 	// Insecure allows insecure connections to the registry when set to true.
 	Insecure bool `json:"insecure,omitempty"`
+	// Platforms allows to specify the list of platform to scan.
+	// If not set, the catalog will be triggered to list all the available platforms on the registry.
```

Comment:
Given this is going to be shown to end users, I would be a bit more generic:

```suggestion
	// If not set, all the available platforms of a container image will be scanned.
```